ArdalisSpecification/src/Ardalis.Specification/IRepositoryBase.cs

```diff
@@ -10,7 +10,7 @@ namespace Ardalis.Specification
     /// </para>
     /// </summary>
     /// <typeparam name="T">The type of entity being operated on by this repository.</typeparam>
```

Comment:
Remove `<typeparam>`

Reply:
I didn't pay attention to the doc comments. We'll have to cross check everything in that regard. I'll remove this one now.